qutebrowser/config/configtypes.py

```diff
@@ -806,19 +819,21 @@ def validate(self, value):
                 raise configexc.ValidationError(value, "may not be empty!")
         value = os.path.expanduser(value)
         try:
-            if not os.path.isfile(value):
-                raise configexc.ValidationError(value, "must be a valid file!")
             if not os.path.isabs(value):
+                cfgdir = standarddir.config()
+                if cfgdir is not None and os.path.isfile(
+                        os.path.join(cfgdir, value)):
+                    return
                 raise configexc.ValidationError(
-                    value, "must be an absolute path!")
+                    value, "must be an absolute path when not using a config "
+                    "directory!")
```

Comment:
This would also be raised if `cfgdir` is `None` but the file doesn't exist - I'd say something like this would make more sense:

```python
if cfgdir is None:
    raise configexc.ValidationError(
                    value, "must be an absolute path when not using a config "
                    "directory!")
if not os.path.isfile(os.path.join(cfgdir, value)):
    raise configexc.ValidationError(
                    value, "must be a valid path relative to the config "
                    "directory!")
```